spec/druuid_spec.rb

```diff
@@ -54,4 +59,14 @@
       after { Druuid.epoch = @old_epoch }
     end
   end
+
+  describe '.min_for_time' do
+    it_behaves_like 'druuid generator', :gen
+
+    let(:datetime) { Time.utc 2012, 2, 4, 8 }
+    it 'generates druuid without random bits' do
+      random_bits = Druuid.min_for_time(Time.now) & Integer("0b"+"1"*Druuid::NUM_RANDOM_BITS)
```

Comment:
can we ditch the Time.now and use the fixed datetime so we can compare to a fixed value?